inc/wpseo-non-ajax-functions.php

```diff
@@ -90,11 +90,20 @@ function wpseo_admin_bar_menu() {
 			}
 
 			if ( $new_notifications_count ) {
-				$notification = sprintf(
-					/* translators: %d resolves to the number of alerts being added. */
-					_n( 'You have a new issue concerning your SEO!', 'You have %d new issues concerning your SEO!', $new_notifications_count, 'wordpress-seo' ),
-					$new_notifications_count
-				);
+				if ( $new_notifications_count === 1 ) {
+					$notification = sprintf(
+						/* translators: %d resolves to the number of alerts being added. */
```

Comment:
You are - of course - completely correct ;-) Too quick with the copy & paste. Fixed now.